bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,290 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [System Requirements](#system-requirements)
+2. [Install Babylon Binary](#install-babylon-binary)
+3. [Setup Node Home Directory and Configure](#setup-your-node-home-directory-and-configure)
+4. [Setup Required Keys](#setup-the-required-keys-for-operating-a-validator)
+5. [Sync Node](#sync-node)
+6. [Get Funds](#get-funds)
+7. [Create Validator](#create-validator)
+
+## System Requirements
+
+The Babylon node is built using the Cosmos SDK and has similar system requirements with typical Cosmos SDK ecosystem nodes.
+
+As a reference, the following system specification is typically used by validators and found to be comfortable for high up-time operations.
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
+- 100MBps bidirectional internet connection
+
+Note: the above is a sample system specification that might not fit your specific infrastructure. Please do your own research before committing to a
+setup and verifying that it can scale well depending on your operations and needs.
+
+## Install Babylon Binary 
+<!-- TODO: check add in the correct tag for the testnet -->
+Download [Golang 1.21](https://go.dev/dl) 
+
+Using the go version 1.21.x (where x is any patch version like 1.21.0, 1.21.1, etc.)
+Once installed run: 
+
+```shell
+go version
+```
+
+To ensure you are using the correct version, the terminal should return something similar to the below.
+
+```shell
+go: downloading go1.21 (darwin/arm64)
+```
```

Comment:
I'd remove this one, the above command already confirms that we're using the correct version.